src/components/modal/Modal.scss

```diff
@@ -0,0 +1,63 @@
+body.modal-visible {
+  height: 100vh;
+  overflow-y: hidden !important;
+}
+
+.modal-backdrop {
+  background: var(--modal-backdrop-background-color);
+  height: 100%;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 500;
+}
+
+.modal-wrapper {
+  left: 50%;
+  outline: 0;
+  position: fixed;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: inherit;
+  z-index: 700;
+}
+
+.modal {
+  background: white;
+  border-top: 10px solid var(--modal-border-color);
+  position: relative;
+  margin: auto;
+  padding: var(--spacing-s) var(--spacing-m) var(--spacing-m);
+  min-width: 450px;
+  width: 100%;
+  z-index: 100;
+}
+
+.modal-header {
+  align-items: center;
+  display: flex;
+}
+
+.modal-title {
+  margin-top: 0;
+  margin-bottom: 0;
+  margin-left: var(--spacing-s);
+}
+
+.modal-close {
+  color: var(--modal-close-button-color);
+  margin-left: auto;
+}
+
+.modal-content {
+  line-height: var(--lineheight-l);
+  margin-top: var(--spacing-s);
+}
+
+.modal-actions {
+  margin-top: var(--spacing-xl);
+  display: grid;
```

Comment:
Grid for displaying two buttons side by side seems little overkill? It feels like there's an easier way to do this, without introducing the complex concept of grid here? 🤔

Reply:
Maybe 🤔 😁  Basic grid is not complex at all but I'll check if there is other way to do this.
Notice that the buttons width was in design 50% - margin in between (no matter how long the content id)